src/css-translation.js

```diff
@@ -61,10 +62,78 @@
 
     this.matrix = computeMatrix(this);
     this.is2D = this.matrix.is2D;
-    this.cssText = generateCssString(this);
+
+    Object.defineProperty(this, 'cssText', {
+      get: function() {
+        if (!this._cssText) {
+          this._cssText = generateCssString(this);
+        }
+        return this._cssText;
+      },
+      set: function(newCssText) {}
+    });
   }
   internal.inherit(CSSTranslation, internal.CSSTransformComponent);
 
+  // These functions (cssTranslationFromTranslate*) are for making CSSTranslations from parsed CSS
+  // Strings. These are needed for setting the cssText.
+  function cssTranslationFromTranslate(coords, string, remaining) {
```

Comment:
This wasn't what I was thinking either - now you've moved parsing logic into the main file. You can definitely keep them separate - I'll send you an example in a bit.